handbook: add a note on reducing image sizes before uploading to Google Cloud. (#5831)
sourcegraph/handbook: content/departments/engineering/dev/process/product_documentation.md

1. Always cross-link to `.md` files, including the file extension, so that the docs are browsable as-is (e.g., in GitHub's file browser).
1. When you create a new directory, always start with an `index.md` file. Don't use another file name and don't create `README.md` files.
1. Don't use special chars and spaces, or capital letters in file names, directory names, branch names, and anything that generates a path.
1. When creating a new document, and it has more than one word in its name, use underscores instead of spaces or dashes (`-`). For example, a proper name would be `import_projects_from_github.md`.

### Images and binary assets

For large images and other binary assets, upload them to the `sourcegraph-assets` Google Cloud Storage bucket instead with `gsutil cp -a public-read local/path/to/myasset.png gs://sourcegraph-assets/` (and refer to them as `https://sourcegraphstatic.com/myasset.png`). For a more detailed instructions visit [this page](../../../../handbook/editing/handbook-images-video.md).

> **Important: make sure to use [ImageOptim.app](https://imageoptim.com/mac) to reduce pictures size as large picture sizes degrade page loading speed.**

Reviewer: ```suggestion
**Important: make sure to use [ImageOptim.app](https://imageoptim.com/mac) to reduce the size of the images before uploading, since large images degrade page loading speed.**
```
